src/TLuaInterpreter.cpp

```diff
@@ -8369,7 +8369,7 @@ int TLuaInterpreter::getScript(lua_State* L)
     auto ids = host.getScriptUnit()->findScriptId(name);
     auto pS = host.getScriptUnit()->getScript(ids.value(pos, -1));
```

Comment:
:information_source: If you remove the `pos--` on line 8367 you can use the *more efficient* `--pos` on THIS line...